tests/phpunit/tests/cache.php

```diff
@@ -378,7 +378,7 @@ public function test_wp_cache_set_multiple() {
 	/**
 	 * @ticket 20875
 	 */
-	public function test_wp_cache_get_multiple() {
+	public function test_get_multiple() {
```

Comment:
Let's not change unrelated test names in this PR.

Reply:
reverted